awswrangler/s3/_read_text.py

```diff
@@ -171,25 +140,40 @@ def _read_text(
     version_id_dict = {path: _get_version_id_for(version_id, path) for path in paths}
 
     if chunksize is not None:
```

Comment:
So I have been having this debate on whether using the `chunksize` argument makes sense in the `distributed` scenario or not. In the current implementation of [read_parquet](https://github.com/aws/aws-sdk-pandas/blob/release-3.0.0/awswrangler/s3/_read_parquet.py#L326), chunksize is never reached because it would always hit the first condition first (i.e. config.distributed == True).

My rationale was that if you are trying to read distributed, then you want to leverage ray datasets and reading in chunk does not make sense. But based on the debate we've had with Anton on making decisions on behalf of the user, it's probably not a fair assumption. So I will follow the same order of conditions in read_parquet too, thanks